docs/python/reference/cli.md

```diff
@@ -63,63 +11,49 @@ Execute a DBOS application in debug mode to replay a specified workflow.
 List workflows run by your application in JSON format ordered by recency (most recently started workflows last).
 
 **Arguments:**
-- `-n, --name <string>`                 Retrieve functions with this name
-- `-l, --limit <number>`                Limit the results returned (default: "10") 
-- `-u, --user <string>`                 Retrieve workflows run by this user
-- `-s, --start-time <string>`           Retrieve workflows starting after this timestamp (ISO 8601 format)
-- `-e, --end-time <string>`             Retrieve workflows starting before this timestamp (ISO 8601 format)
-- `-S, --status <string>`               Retrieve workflows with this status (`PENDING`, `SUCCESS`, `ERROR`, `RETRIES_EXCEEDED`, `ENQUEUED`, or `CANCELLED`)
-- `-v, --application-version <string>`  Retrieve workflows with this application version
-- `--request`                           Retrieve workflow request information
+* `-D, --db-url TEXT`: Your DBOS application database URL
+* `-l, --limit INTEGER`: Limit the results returned  [default: 10]
+* `-u, --user TEXT`: Retrieve workflows run by this user
+* `-s, --start-time TEXT`: Retrieve workflows starting after this timestamp (ISO 8601 format)
+* `-e, --end-time TEXT`: Retrieve workflows starting before this timestamp (ISO 8601 format)
+* `-S, --status TEXT`: Retrieve workflows with this status (PENDING, SUCCESS, ERROR, RETRIES_EXCEEDED, ENQUEUED, or CANCELLED)
+* `-v, --application-version TEXT`: Retrieve workflows with this application version
+* `-n, --name TEXT`: Retrieve workflows with this name
+* `-d, --sort-desc`: Sort the results in descending order (older first)
+* `-o, --offset INTEGER`: Offset for pagination
```

Comment:
these are simply taken from the output of `python3 -m typer dbos.cli.cli utils docs`